upcase_booklist_app/templates/category.html

```diff
@@ -0,0 +1,37 @@
+<html>
+    <head>
+        <title>Curious Programmer</title>
```

Comment:
You should start adding the subpage to the title of each page, such as `Curious Programmer | Categories`.